doc/source/examples/rpt/photon-count-calculation-in-a-cylindrical-vessel/photon-count-calculation-in-a-cylindrical-vessel.rst

```diff
@@ -0,0 +1,325 @@
+
+Scenario 2: Horizontal translation of a particle along the y-axis 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. figure:: images/result_2a.png
+    :alt: Scenario 2 results when reactor attenuation coefficient is set at 10
+    :align: center
+    :name: Results of the 2nd scenario, case I
+
+    Case I: :math:`\mu_r = 10, \ \mu_d = 21.477`
+
+:raw-html:`<br />`
+
+
+The figure shown above illustrates the photon count of the particle as it travels from the back to the front of the vessel along the y-axis. The Case I figure shows the evolution of the photon count for the system we are currently studying (:math:`\mu_r = 10, \ \mu_d = 21.477`). Let's analyze the resulting plot.
+
+First, a symmetry of photon counts from the center axis of the detector can be seen. Such symmetry should be expected since the detector is symmetrical from its center axis.
+
+Secondly, we can notice that the variation in photon count as the particle travel is quite small. The difference between its maximal and minimal values is approximately :math:`147`, which is one order of magnitude smaller than the other scenarios. This may mainly be explained by the small variations in the distance between the particle and the detector's exposed surface. In other words, the lengths of the paths traveled by the photon in the vessel and in the detector vary less than in the other scenarios.
+
+Lastly, as the particle travels across the vessel, we notice fluctuations in the photon count. Starting from the back of the vessel, the photon count decreases rapidly until a local minimal value at approximately :math:`y = -6` cm and then increases until a local maximum at :math:`y = 0` cm (center of the detector's face). Then, from the center to the front of the vessel, a mirrored image of the photon count's evolution can be seen. To understand the fluctuations, let's look at three other figures (Case II, Case III, and Case IV) while focusing on the first half of the studied domain (:math:`y \in ]-10, 0]` cm) since the evolution of the count is symmetrical from :math:`y = 0` cm.
+
++---------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------+
+|  .. figure:: images/result_2b.png                                                                       |  .. figure:: images/result_2c.png                                                                       |
+|    :alt: Scenario 2 results when the efficiency of the detector is the product of the weighting factors |     :alt: Scenario 2 results when reactor attenuation coefficient is set at 0                           |
```

Comment:
When the efficiency is only a function of weighting factors and fa , fd have been intentionally considered equal at all positions.

We should mark it that this is a study case, and we fixed fa and fd to not change.